modules/microshift-configuring-ovn.adoc

```diff
@@ -76,8 +97,3 @@ mtu: 1300
 ====
 When `disableOVSInit` is set to true in the `ovn.yaml` config file, the OVS bridge br-ex must be manually configured.
 ====
-
```

Comment:
Should this note be indented since it's applying to an item in the table? I understand why it's not a footnote so it doesn't get lost but maybe the indent would make it a bit more clear that it applies to an item in the table.